lib/tasks/build.js

```diff
@@ -17,6 +18,7 @@ class BuildTask extends Task {
       outputPath: options.outputPath,
       environment: options.environment,
       project: this.project,
+      broccoliMaster: experiments.BROCCOLI_MASTER,
```

Comment:
Should be able to revert changes to this file once the builder itself does its own experiments check...